string getTime

int32 fixedstate

float64 latitude
float64 longitude
float64 altitude

float64 utm_x
float64 utm_y

float64 heading_rad
float64 heading_pi
float64 heading_degrees

float64 position_error